backend/core/helpers.py

```
@@ -927,7 +927,7 @@ def viewable_items(model, folder_id=None):
     viewable_requirement_assessments = viewable_items(RequirementAssessment, folder_id)
     controls_count = viewable_controls.count()
     progress_avg = math.ceil(
-        mean([x.progress for x in viewable_compliance_assessments] or [0])
+        mean([x.get_progress() for x in viewable_items(ComplianceAssessment)] or [0])
```

Ruff (0.8.2)

`ComplianceAssessment` may be undefined, or defined from star imports

(F405)